src/FileMigrationHelper.php

```diff
@@ -143,12 +155,33 @@ protected function migrateFile($base, File $file, $legacyFilename)
             $file->setFilename($result['Filename']);
         }
 
+        // set original 'LastEdited' date if static is enabled
+        $lastEditedDate = '';
+        if ($this->config()->get('keep_lastedited_date')) {
+            $lastEditedDate = $file->LastEdited;
+        }
+
         // Save and publish
         $file->write();
         if (class_exists(Versioned::class)) {
             $file->copyVersionToStage(Versioned::DRAFT, Versioned::LIVE);
         }
 
+        // if 'LastEdited' date static is enabled, update file's last edited using SQL query
+        if ($this->config()->get('keep_lastedited_date') && $lastEditedDate) {
+            // Draft table
+            $table = DataObject::singleton(File::class)->baseTable();
```

Comment:
Those updates need to have columns and tables quoted, otherwise Postgres with crap itself